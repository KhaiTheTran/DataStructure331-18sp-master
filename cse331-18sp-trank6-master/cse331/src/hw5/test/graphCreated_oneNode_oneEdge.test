# Test graphCreated_oneNode_oneEdge

# Create a graph
CreateGraph graphCreated_oneNode_oneEdge

# Add a node
AddNode graphCreated_oneNode_oneEdge xx

# Add an edge
AddEdge graphCreated_oneNode_oneEdge xx xx xxxx

# Print the nodes in the graph and the outgoing edges from xx
ListNodes graphCreated_oneNode_oneEdge
ListChildren graphCreated_oneNode_oneEdge xx