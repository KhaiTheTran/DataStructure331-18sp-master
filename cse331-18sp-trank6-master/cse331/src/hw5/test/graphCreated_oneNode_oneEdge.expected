# Test graphCreated_oneNode_oneEdge

# Create a graph
created graph graphCreated_oneNode_oneEdge

# Add a node
added node xx to graphCreated_oneNode_oneEdge

# Add an edge
added edge xxxx from xx to xx in graphCreated_oneNode_oneEdge

# Print the nodes in the graph and the outgoing edges from xx
graphCreated_oneNode_oneEdge contains: xx
the children of xx in graphCreated_oneNode_oneEdge are: xx(xxxx)